### DISCUSSION TITLE
需要支持在其它链上发布挑战吗

### DISCUSSION THREAD

**Discussion by deanplus** [decert-me/website#281] [General]
## 讨论项
1. 需要支持在其它链上发布挑战吗？还是只领取（空投）到其它链？还是都支持？
    可以在这里投票和评论：https://github.com/orgs/decert-me/discussions/293
2. 支持的话，各条链的优先级别
    待定
 

## 现状：
当前发布挑战是在 Polygon，领取（空投）挑战 NFT 则可以在 Polygon 和 Solana（根据用户当前连接到哪条链）

**Comment by xilibi2003** [comment]
支持多链部署， 发布挑战也支持多链吧

**Reply by xilibi2003** [reply]
优先支持 Op 和 Arb
